client/src/about/about.js

```diff
@@ -1,13 +1,23 @@
 import _ from "lodash";
 import React from "react";
 
+import {
+  IconHTML,
+  IconNodeJS,
+  IconReact,
+  IconGit,
+  IconGitHub,
+  IconPython,
+  IconSass,
+  IconGraphQL,
+  IconBaselineCloud,
+} from "src/icons/icons";
```

Comment:
Minor note, will eventually add a linter for this probably, but please use the full file name including extension when importing. Less ambiguous and can also have an effect on the automatic import order rule in some cases, so good to be consistent about.